app/api/grape_api/root_api.rb

```diff
@@ -0,0 +1,98 @@
+      expose :current_sign_in_at, documentation: { type: DateTime }
+      expose :last_sign_in_at, documentation: { type: DateTime }
+      expose :current_sign_in_ip, :last_sign_in_ip
+      expose :created_at, :updated_at, documentation: { type: DateTime }
+      expose :admin, :enabled, documentation: { type: "boolean" }
+      expose :ldap_name, documentation: { type: String, desc: "Ldap name" }
+      expose :failed_attempts, documentation: { type: Integer }
+      expose :locked_at, documentation: { type: DateTime }
+      expose :namespace_id, documentation: { type: Integer }
+      expose :display_name, documentation: { type: String, desc: "Display name" }
+    end
+
+    class ApplicationTokens < Grape::Entity
+      expose :id, unless: { type: :create }, documentation: { type: Integer }
+      expose :application, unless: { type: :create }
+      expose :plain_token, if: { type: :create }
+    end
+
+    class ApiErrors < Grape::Entity
+      expose :errors, documentation: {
+        type: "GrapeAPI::Entities::Messages", is_array: true
+      }
+    end
+
+    class Messages < Grape::Entity
+      expose :message
+    end
+  end
+
+  class RootAPI < Grape::API
+    # version 'v1', using: :path
+    format :json
+    prefix :api
+
+    before do
+      header["Access-Control-Allow-Origin"] = "*"
+      header["Access-Control-Request-Method"] = "*"
+      header["Access-Control-Allow-Headers"] = "Content-Type, api_key, Authorization, portus-auth"
+    end
+
+    rescue_from ActiveRecord::RecordNotFound do
+      error_response message: "Not found", status: 404
+    end
+
+    rescue_from Grape::Exceptions::ValidationErrors do |e|
+      error_response message: { errors: e.errors }, status: 400
+    end
+
+    # global exception handler, used for error notifications
+    rescue_from :all do |e|
+      error_response message: "Internal server error: #{e}", status: 500
+    end
+
+    helpers do
+      def authenticate!
+        return if request.request_method == "OPTIONS"
+        error!("Authentication fails.", 401) unless (user = current_user)
+        error!("Authorization fails.", 403) unless user.admin
+      end
+
+      def current_user
+        auth = request.headers["Portus-Auth"].presence
+        return if auth.nil?
+        username, password = auth.split(":")
+        @user = User.find_by(username: username)
+        @user if @user && @user.application_token_valid?(password)
+      end
+    end
+
+    mount GrapeAPI::UserAPI
+
+    add_swagger_documentation \
+      security_definitions: {
+        api_key: {
+          type: "apiKey",
+          name: "Portus-Auth",
+          in:   "header"
+        }
+      },
+      security:             [api_key: []],
+      info:                 {
+        title:         "Portus user API",
+        description:   "CRUD users and user's application tokens API",
+        contact_name:  "Anldrei Kislichenko",
+        contact_email: "andrei.kislichenko@gmail.com"
```

Comment:
I'd prefer:

- Contact name: "Portus authors" (which would include you 😉)
- Contact email: "portus-dev@googlegroups.com"